File: web3rpc/rpc-specs/code-samples/javascript/klay/block/getCommitteeSize.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const blockNumberOrTag = "0x1b4";

  provider.klay
    .getCommitteeSize({ blockNumberOrTag }, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/kaia/block/getCommitteeSize.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

blockTag = "0x1b4"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.get_committee_size(blockTag)

print(kaia_response)
